test/src/tests-serverModel.ts

```diff
@@ -0,0 +1,1675 @@
+                    return '123456';
+                },
+                getConsentState: () => {
+                    return null;
+                },
+            };
+
+            const event = {
+                MPID: '555666777',
+            };
+
+            ServerModel.appendUserInfo(user, event as SDKEvent);
+
+            expect(event.MPID).to.equal('123456');
+        });
+
+        it('returns early if event and user are not valid', () => {
+            expect(ServerModel.appendUserInfo(null, null)).to.be.undefined;
+        });
+    });
+
+    describe('#convertToConsentStateDTO', () => {
+        it('should convert Consente State with GDPR to a DTO', () => {
+            const consentState = ({
+                getGDPRConsentState: () => {
+                    return {
+                        'test-gdpr-purpose': {
+                            Consented: true,
+                            Timestamp: 12345,
+                            ConsentDocument: 'gdpr-doc',
+                            Location: 'test-gdpr-location',
+                            HardwareId: 'test-gdpr-hardware-id',
+                        },
+                    };
+                },
+                getCCPAConsentState: () => {},
+            } as unknown) as SDKConsentState;
+            const expectedDTO = {
+                gdpr: {
+                    'test-gdpr-purpose': {
+                        c: true,
+                        d: 'gdpr-doc',
+                        h: 'test-gdpr-hardware-id',
+                        l: 'test-gdpr-location',
+                        ts: 12345,
+                    },
+                },
+            };
+
+            expect(ServerModel.convertToConsentStateDTO(consentState)).to.eql(
+                expectedDTO
+            );
+        });
+
+        it('should convert Consente State with CCPA to a DTO', () => {
+            const consentState = ({
+                getGDPRConsentState: () => {},
+                getCCPAConsentState: () => {
+                    return {
+                        Consented: false,
+                        Timestamp: 67890,
+                        ConsentDocument: 'ccpa-doc',
+                        Location: 'test-ccpa-location',
+                        HardwareId: 'test-ccpa-hardware-id',
+                    };
+                },
+            } as unknown) as SDKConsentState;
+            const expectedDTO = {
+                ccpa: {
+                    data_sale_opt_out: {
+                        c: false,
+                        d: 'ccpa-doc',
+                        h: 'test-ccpa-hardware-id',
+                        l: 'test-ccpa-location',
+                        ts: 67890,
+                    },
+                },
+            };
+
+            expect(ServerModel.convertToConsentStateDTO(consentState)).to.eql(
+                expectedDTO
+            );
+        });
+
+        it('should convert Consente State with both GDPR and CCPA to a DTO', () => {
```

Comment:
global find and replace for `Consente` --> `Consent`

Reply:
Good catch